azure_arc_servers_jumpstart/gcp/windows/main.tf

```diff
@@ -0,0 +1,86 @@
+// Terraform plugin for creating random ids
+resource "random_id" "instance_id" {
+ byte_length = 8
+}
+
+// An Azure Resource Group
+resource "azurerm_resource_group" "azure_rg" {
+  name = var.azure_resource_group
+  location = var.azure_location
+}
+
+// A single Google Cloud Engine instance
+resource "google_compute_instance" "default" {
+ name = "azure-arc-gcp-demo-${random_id.instance_id.hex}"
```

Comment:
I think the name is too long. Also, consider the 15 characters' Windows hostname limit. Can drop the "azure" IMO.